01-data-structures/02-stacks-and-queues/myqueue/myqueue.rb

```diff
@@ -8,11 +8,21 @@ def initialize
   end
 
   def enqueue(element)
+    @queue = [element] + @queue;
+    @head = @queue[0];
+    self.tail = @queue.last;
   end
 
   def dequeue
+    @queue.delete(@queue[0]);
+    @head = @queue[0];
   end
 
   def empty?
+    if @queue.length < 1
+      true;
+    else
+      false;
+    end
```

Comment:
This can be simplified to just `@queue.length < 1`, since that will be true or false anyway.